Sources/Styleguide/DesignSystems/Cursor.swift

```diff
@@ -0,0 +1,11 @@
+import Css
+import Prelude
+
+extension Class {
+  public enum cursor {
+    public static let pointer = CssSelector.class("cursor-pointer")
+  }
+}
+
+let cursorStyles =
+  Class.cursor.pointer % key("cursor", "pointer")
```

Comment:
got to use this to make `input[type=submit]` html elements show cursor pointer on rollover. we want this for the underline button cause otherwise it doesnt look super clickable.